tests/fast-integration/logging/loki.js

```diff
@@ -50,7 +50,7 @@ function parseJson(str) {
 }
 
 async function verifyIstioAccessLogFormat(startTimestamp) {
-  const query = '{container="istio-proxy",job="fluent-bit",namespace="kyma-system",pod="logging-loki-0"}';
```

Comment:
telemetry-tests should have even more queries with job="fluent-bit"

Reply:
The telemetry-tests explicitly query for job="telemetry-fluent-bit". Thought about adding the label here as well. Two reasons against that are:
1. We want to test logging in an end-to-end manner and not the telemetry-fluent-bit in this test
2. Having the telemetry-fluent-bit label here would break upgrade tests

Reply:
Yes, was asking because telemetry component is default now if we even need the explicit query on job in telemetry tests anymore.

Reply:
Removed the job label from the telemetry test.